presto-native-execution/src/test/java/com/facebook/presto/nativeworker/PrestoNativeQueryRunnerUtils.java

```diff
@@ -151,6 +151,11 @@ public static QueryRunner createNativeQueryRunner(
             boolean addStorageFormatToPath)
             throws Exception
     {
+        ImmutableMap.Builder<String, String> hivePropertiesBuilder = new ImmutableMap.Builder<>();
+        hivePropertiesBuilder
+                .putAll(getNativeWorkerHiveProperties(storageFormat))
+                .put("hive.allow-drop-table", "true");
```

Comment:
Why is this not part of getNativeWorkerHiveProperties ?

Can you update PR description to explain why you are making this change?

Reply:
+1. Agree with @mbasmanova. I too feel this should be a part of getNativeWorkerHiveProperties(). Several people look at that function when setting up their catalogs.

Reply:
We tried that. `getNativeWorkerHiveProperties()` is used on the Spark side as well and it conflicted.
 The diff would look like https://github.com/prestodb/presto/commit/2affa784e3f7f2070d3d149698591d4b043629cf

You will see that on the Spark side, the corresponding config is set inside the PrestoSparkQueryRunner.
We followed the existing pattern and avoided risky refactoring.

Reply:
@mbasmanova I updated the description. Thanks.

Reply:
Since these classes are used for both Native and Java paths, we should have native-only, java-only, and common config.
`hive.allow-drop-table` is a common config.

Reply:
If the above classification makes sense, we could refactor the existing config in a separate PR.

Reply:
I guess there is also Spark config.

Reply:
Do we know why PrestoSparkQueryRunner sets this by default, but Hive query runner doesn't?

CC: @arhimondr